config/robots.rb

```diff
@@ -2940,4 +2940,103 @@
         }
       }
     )
+
+    # LCMB Bravo bed verification
+    # LCMB Cherrypick to LCMB End Prep
+    # Transfers 1:1
+    custom_robot(
+      'bravo-lcmb-cherrypick-to-lcmb-end-prep',
+      name: 'Bravo LCMB Cherrypick => LCMB End Prep',
+      beds: {
+        bed(4).barcode => {
+          purpose: 'LCMB Cherrypick',
+          states: ['passed'],
+          label: 'Bed 4'
+        },
+        car('1,4').barcode => {
+          purpose: 'LCMB End Prep',
+          states: ['pending'],
+          label: 'Carousel 1,4',
+          parent: bed(4).barcode,
+          target_state: 'started'
+        }
+      }
+    )
+
+    # LCMB Bravo bed verification
+    # Checks LCMB End Prep plate is on deck (alone)
+    # Requires robot scan so can be checked in next bed verification
+    custom_robot(
+      'bravo-lcmb-end-prep',
+      name: 'Bravo LCMB End Prep',
+      require_robot: true,
+      beds: {
+        bed(7).barcode => {
+          purpose: 'LCMB End Prep',
+          states: ['started'],
+          label: 'Bed 7',
+          target_state: 'passed'
+        }
+      }
+    )
+
+    # LCMB Bravo bed verification
+    # LCMB End Prep to LCMB Lib PCR
+    # Transfers 1:1
+    # Checks same robot used as for previous bed verification ie. LCMB End Prep is in same
+    # position on same robot as before.
+    custom_robot(
+      'bravo-lcmb-end-prep-to-lcmb-lib-pcr',
+      name: 'Bravo LCMB End Prep => LCMB Lib PCR',
+      verify_robot: true,
+      beds: {
+        bed(7).barcode => {
+          purpose: 'LCMB End Prep',
+          states: ['passed'],
+          label: 'Bed 7'
+        },
+        bed(6).barcode => {
+          purpose: 'LCMB Lib PCR',
+          states: ['pending'],
+          label: 'Bed 6',
+          parent: bed(7).barcode,
+          target_state: 'passed'
+        }
+      }
+    )
+
+    # LCMB Hamilton Star bed verification
+    # LCMB Lib PCR to LCMB Lib PCR-XP
+    # Transfers up to 2 pairs of plates at the same time
+    custom_robot(
+      'star-96-lcmb-lib-pcr-purification',
+      name: 'STAR-96 LCMB Lib PCR => LCMB Lib PCR-XP',
+      verify_robot: false,
+      beds: {
+        bed(7).barcode => {
+          purpose: 'LCMB Lib PCR',
+          states: ['passed'],
+          label: 'Bed 7'
+        },
+        bed(9).barcode => {
+          purpose: 'LCMB Lib PCR-XP',
+          states: ['pending'],
+          label: 'Bed 9',
+          parent: bed(7).barcode,
+          target_state: 'passed'
+        },
+        bed(12).barcode => {
+          purpose: 'LCMB Lib PCR',
+          states: ['passed'],
+          label: 'Bed 12'
+        },
+        bed(14).barcode => {
+          purpose: 'LCMB Lib PCR-XP',
+          states: ['pending'],
+          label: 'Bed 14',
+          parent: bed(12).barcode,
+          target_state: 'passed'
+        }
+      }
+    )
```

Comment:
Does 7 always go to 9, and 12 to 14. Or could 7 go to 14 and 12 to 9?

Reply:
They are matched in pairs, 7 must go with 9, and 12 with 14, it's not that flexible that you can use other combinations. But you can use just one pair and leave the other pair empty, the method on the robot allows for transferring either one set of plates or two.